### ISSUE TITLE
gtburst fails to launch in alpha

### ISSUE THREAD

**Issue by donhorner** [fermi-lat/pyBurstAnalysisGUI#5]
If I try to run gtburst in the latest alpha, it fails with 

```(fermi-dev) [dhorner@gs66-gloom] ~ % gtburst

Loading pyBurstAnalysisGUI v. 02-03-00p20 .....Traceback (most recent call last):
  File "/Users/dhorner/anaconda2/envs/fermi-dev/lib/python2.7/site-packages/fermitools/gtburst.py", line 67, in <module>
    from GtBurst import dataHandling
  File "/Users/dhorner/anaconda2/envs/fermi-dev/lib/python2.7/site-packages/fermitools/GtBurst/dataHandling.py", line 16, in <module>
    import BinnedAnalysis
  File "/Users/dhorner/anaconda2/envs/fermi-dev/lib/python2.7/site-packages/fermitools/BinnedAnalysis.py", line 326
    config=config)
    ^
IndentationError: unexpected indent
```
Do you also see this behavior @dagorym?

**Comment by jasercion** [comment]
Fix [committed](https://github.com/fermi-lat/pyLikelihood/commit/51c3f1120f3fe15b704aefe29e1cb8c0ca5c6b18).  This is actually an issue with pyLikelihood.  I'm building and testing the fix now.

**Comment by jasercion** [comment]
Fixed as of version Fermitools=1.1.2.  Closing issue.
